### PR TITLE
Corrected html not showing up correctly in quest modal

### PR DESCRIPTION
**Pull request by crookedneighbor**
screen shot 2014-10-17 at 12 46 08 pm

### CODE REVIEW

File: views/shared/modals/quests.jade [hunk 1]
```diff
@@ -5,7 +5,7 @@ script(type='text/ng-template', id='modals/questCompleted.html')
     h4 "{{::Content.quests[user.party.quest.completed].text()}}"
       =env.t('completed')
   .modal-body
-    p {{::Content.quests[user.party.quest.completed].completion()}}
+    p(ng-bind-html='::Content.quests[user.party.quest.completed].completion()') {{::Content.quests[user.party.quest.completed].completion()}}
```
Comment (paglias) [inline views/shared/modals/quests.jade]: ok! but you should remove the old binding {{::Content.quests[user.party.quest.completed].completion()}}

**Comment by crookedneighbor** [comment]
@paglias You're totally right. Fixed.

**Comment by crookedneighbor** [comment]
Forgot to include this: fixes #4130

**Comment by paglias** [comment]
@crookedneighbor merged. I'm not sure who has awarded you the previous levels but I leave the decision to award you an additional level to them

Maybe @deilann or @Alys ?

**Comment by crookedneighbor** [comment]
UUID: `75f270e8-c5db-4722-a5e6-a83f1b23f76b`

**Comment by paglias** [comment]
@crookedneighbor level 3 awarded! for this and #4161

**Pull request merged**